CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/release_notes/index.html.
   Full Keycloak upstream jira issue can be shown if filtered by Fix version. For example [Keycloak jira issue for 15.0.2 version](https://issues.redhat.com/browse/KEYCLOAK-19161?jql=project%20%3D%20keycloak%20and%20fixVersion%20%3D%2015.0.2)
 
+
+## [5.8.0] - 2025-
+
+### Added
+- React in Account Console
+- Preffered Username in Profile Paga 
```

Comment:
Page

Reply:
Do we need to introduce a new `preferredUsername` message? Don't we already define `uid` in the vanilla theme?

Reply:
> Do we need to introduce a new `preferredUsername` message? Don't we already define `uid` in the vanilla theme?

See my general comment.
We do not have `preferredUsername` message in account messages.

Reply:
Updated msgKyey to use uid and updated CHANGELOG/README. Ready for release.